tools/buildtools/ambertools/tleap.xml

```diff
@@ -0,0 +1,1987 @@
+            <param name="arg_direction" label="direction value in each dimension" type="text" value="" help="Displace by this amount { X Y Z }. The direction is based on whether each vector is positive or negative. "/>
+       </when>
+       <when value="verbosity">
+            <param name="arg_level" label="level value" type="text" value="" help="This command sets the level of output that LEaP provides the user. A value of 0 is the default, providing the minimum of messages. A value of 1 will produce more output, and a value of 2 will produce all of the output of level 1 and display the text of the script lines executed with the source command."/>
+       </when>
+       <when value="zMatrix">
+            <param name="arg_obj" label="obj value" type="text" value="" help="UNIT/RESIDUE/ATOM"/>
+            <param name="arg_zmatrix" label="zmatrix value" type="text" value="" help="LIST"/>
+       </when>
+    </conditional>
+</repeat>
+    </inputs>
+    <outputs>
+        <data format="txt" name="output_tleap_in" label="Build tLEaP: tleap.in"/>
+        <data format="txt" name="output_tleap" label="Build tLEaP: tleap.log"/>
+        <collection name="output_pdb" type="list" label="Collection of pdb" format="pdb">
+               <discover_datasets pattern="(?P&lt;designation&gt;.+)\.pdb" directory="out" format="pdb"/>
+        </collection>
+        <collection name="output_top" type="list" label="Collection of topology files" format="prmtop">
+               <discover_datasets pattern="(?P&lt;designation&gt;.+)\.prmtop" directory="out" format="prmtop"/>
+        </collection>
+        <collection name="output_txt" type="list" label="Collection of txt" format="txt">
+               <discover_datasets pattern="(?P&lt;designation&gt;.+)\.txt" directory="out" format="txt"/>
+        </collection>
+        <collection name="output_coord" type="list" label="Collection of coordinate files" format="inpcrd">
+               <discover_datasets pattern="(?P&lt;designation&gt;.+)\.inpcrd" directory="out" format="inpcrd"/>
+        </collection>
+        <collection name="output_mol2" type="list" label="Collection of mol2" format="mol2">
+               <discover_datasets pattern="(?P&lt;designation&gt;.+)\.mol2" directory="out" format="mol2"/>
+        </collection>
+        <collection name="output_mol3" type="list" label="Collection of mol3" format="mol3">
+               <discover_datasets pattern="(?P&lt;designation&gt;.+)\.mol3" directory="out" format="mol3"/>
+        </collection>
+    </outputs>
+    <tests>
+        <test>
```

Comment:
expected_num_outputs will be helpful here I think

Reply:
corrected